src/askui/models/askui/settings.py

```diff
@@ -40,12 +48,16 @@ def base_url(self) -> str:
 
 
 class AskUiComputerAgentSettings(ComputerAgentSettingsBase):
-    model: str = ModelName.ANTHROPIC__CLAUDE__3_5__SONNET__20241022
+    model: str = ModelName.ANTHROPIC__CLAUDE__SONNET__4__20250514
     askui: AskUiSettings = Field(default_factory=AskUiSettings)
+    thinking: ThinkingSettings | None = None
+    tool_choice: BetaToolChoiceParam | BetaToolChoiceAutoParam = (
+        BetaToolChoiceAutoParam(type="auto")
+    )
```

Comment:
Can't we just move this one level up into `ComputerAgentSettingsBase` as this is also used inside the one of anthropic?